frontend/public/locales/zh/translation.json

```diff
@@ -1,133 +1,189 @@
+  "Create Gateway": "网关主机",
   "Current duration": "当前持续时间",
-  "Current refresh interval: {{refreshInterval}}": "Current refresh interval: {{refreshInterval}}",
+  "Current refresh interval: {{refreshInterval}}": "当前刷新时间间隔: {{refreshInterval}}",
   "Current time range": "当前时间范围",
   "custom": "自定义",
   "Custom": "自定义",
   "Dagre": "Dagre",
-  "Data Plane": "Data Plane",
+  "Data Plane": "数据面板",
   "Dataplane table": "Dataplane table",
   "Debug information": "Debug information",
   "Debug Information": "Debug Information",
   "Debug information has been copied to your clipboard.": "Debug information has been copied to your clipboard.",
   "Debug information was copied to your clipboard, but is outdated now. It could be caused by new data received by auto refresh timers.": "Debug information was copied to your clipboard, but is outdated now. It could be caused by new data received by auto refresh timers.",
   "Degraded": "Degraded",
+  "Delay Percentage": "延迟比例",
+  "Delete": "删除",
+  "Delete Traffic Routing": "删除流量路由规则",
+  "Destination Hosts": "目的主机",
+  "Destination Workload": "目的工作负载",
   "Details in side panel on nodes": "Details in side panel on nodes",
   "Direction": "方向",
   "Direction type is not valid.": "Direction type is not valid.",
+  "Disable Auto Injection": "禁用自动注入",
   "Disabled": "禁用",
-  "Display": "Display",
-  "Display the legend to learn about what the different shapes, colors and backgrounds mean.": "Display the legend to learn about what the different shapes, colors and backgrounds mean.",
+  "Display": "展示",
+  "Display the legend to learn about what the different shapes, colors and backgrounds mean.": "显示图例以了解不同形状、颜色和背景的含义。",
   "Distributed Tracing": "分布式跟踪",
-  "Done": "Done",
-  "Download": "Download",
+  "Done": "完成",
+  "Download": "下载",
   "Download all resources in a file": "Download all resources in a file",
   "Download configuration in a file": "Download configuration in a file",
-  "Duration": "Duration",
-  "Edges": "Edges",
+  "Duration": "持续时间",
+  "Edges": "边",
+  "Edit Annotations": "编辑Annotations",
+  "Enable Auto Injection": "启动自动注入",
   "Enabled": "启用",
   "Error fetching Istio deployment status.": "Error fetching Istio deployment status.",
+  "Evenly distribute traffic": "平均分配网络流量",
   "Every 10s": "每 10s",
   "Every 15m": "每 15m",
   "Every 15s": "每 15s",
   "Every 1m": "每 1m",
   "Every 30s": "每 30s",
   "Every 5m": "每 5m",
   "Expand view": "展开视图",
-  "Failed Request": "Failed Request",
+  "Failed Request": "失败的请求",
   "Failure": "Failure",
-  "Fault Injection": "Fault Injection",
+  "Fault Injection": "故障注入",
+  "Fault Injection not valid": "故障注入无效",
   "Filter by Application Health": "Filter by Application Health",
   "Filter by mTLS": "Filter by mTLS",
   "Filter by Namespace": "Filter by Namespace",
   "Filter by Namespace Label": "Filter by Namespace Label",
-  "Find and Hide": "Find and Hide",
+  "Find and Hide": "寻找或隐藏",
+  "Fixed Delay": "延迟时长",
   "FQDN": "完全限定域名",
+  "gateway": "网关",
```

Comment:
This entry does not longer exist in the application. Please re-generate the `translation.json`

Reply:
Ok, I re-run yarn build and submitted the updated translation.json.

Reply:
I think I have resolved the conflict. Please review it for me when you have time. If you have any questions, please continue to raise them and I will continue to improve them.